tests/Doctrine/Tests/DBAL/Schema/DB2SchemaManagerTest.php

```diff
@@ -44,7 +45,8 @@ protected function setUp()
      */
     public function testListTableNamesFiltersAssetNamesCorrectly()
     {
-        $this->conn->getConfiguration()->setFilterSchemaAssetsExpression('/^(?!T_)/');
```

Comment:
@bezhermoso please revert this and squash all commits. It looks good to go.

Reply:
@morozov Done! 👍

Reply:
Please squash all commits so that we have a cleaner history.

Reply:
@morozov Squashed